.gitlab-ci/packet.yml

```diff
@@ -115,6 +115,12 @@ packet_centos7-multus-calico:
   extends: .packet
   when: manual
 
+# unit-tests is just for testing
+packet_centos8-calico:
+  stage: unit-tests
```

Comment:
Just a note to remember to revert back to `stage: deploy-part2` before merging/when WIP is finished.